cookbook/controller/error_pages.rst

```diff
@@ -205,86 +187,65 @@ to point to it.
             // ...
         ));
 
-.. tip::
-
-    You can also set up your controller as a service.
-
-    The default value of ``twig.controller.exception:showAction`` refers
-    to the ``showAction`` method of the ``ExceptionController``
-    described previously, which is registered in the DIC as the
-    ``twig.controller.exception`` service.
+The :class:`Symfony\\Component\\HttpKernel\\EventListener\\ExceptionListener`
+class used by TwigBundle as a listener of the ``kernel.exception`` event creates
+the Request that will be dispatched to your controller. In addition, your controller
```

Comment:
I think we should enclose "Request" with literals if we want to refer to the class. Otherwise, it should be lowercased.